docker/ezbake/docker-entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -e
+
+if [ -n "$EZBAKE_REPO" ]; then
```

Comment:
What happens if `$EZBAKE_REPO` is unset? It looks like things will continue, but I'm not sure how that's supposed to work..

Reply:
the build will pull whatever version of ezbake specified in project.clj from clojars/artifactory

Reply:
and yes, that's how it's supposed to work

Reply:
ahhh okay, I see.

And to make sure I'm understanding correctly: if `$PROJECT_REPO` is unset, it just assumes that your project is already mounted into the `/workspace` directory and tries to continue with that?

Reply:
yep!